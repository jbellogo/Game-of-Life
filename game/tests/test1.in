new 5
init
2 1
2 2
2 3
-1 -1
print
step
print
