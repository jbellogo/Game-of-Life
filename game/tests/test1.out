_____
__X__
__X__
__X__
_____
_____
_____
_XXX_
_____
_____
